src/coreComponents/mesh/generators/BuildPods.cpp

```diff
@@ -219,6 +225,8 @@ FaceMgrImpl makeFlavorlessFaceMgrImpl( std::size_t const & numFaces,
                       toFlavorlessMapping( recv ) );
 }
 
+// Function to create a nodeManager from our local upward mapping data
+// By flavorless, we are indicating that we are changing data-structures to GEOS ones, etc
```

Comment:
`flavorless` is there to emphasize that we're no more using the strongly typed integers and fallback to `localIndex` and `globalIndex`.